conans/test/unittests/util/tools_test.py

```diff
@@ -1220,6 +1224,14 @@ def get_values(this_os, this_arch, setting_os, setting_arch, compiler=None):
         self.assertEquals(build, "x86_64-apple-darwin")
         self.assertEquals(host, "aarch64-apple-darwin")
 
+        build, host = get_values("Linux", "x86_64", "Linux", "ppc32")
+        self.assertEquals(build, "x86_64-linux-gnu")
+        self.assertEquals(host, "powerpc-linux-gnu")
+
+        build, host = get_values("Linux", "x86", "Linux", "ppc64")
+        self.assertEquals(build, "x86-linux-gnu")
+        self.assertEquals(host, "powerpc64-linux-gnu")
+
```

Comment:
I suppose it's worth to add test new arches (`armv8.3` & `armv8_32`) resolve to the `arm-apple-darwin` as well.